src/guide/essentials/forms.md

```diff
@@ -442,7 +442,9 @@ If you want user input to be automatically typecast as a number, you can add the
 <input v-model.number="age" />
 ```
 
-If the value cannot be parsed with `parseFloat()`, then the original value is used instead.
+:::warning
```

Comment:
**suggestion** could we please remove the `warning` wrapper here? We're intentionally trying to minimize the number of highlighted blocks

Reply:
Done.